src/boost.mk

```diff
@@ -4,13 +4,14 @@ PKG             := boost
 $(PKG)_WEBSITE  := https://www.boost.org/
 $(PKG)_DESCR    := Boost C++ Library
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 1.60.0
-$(PKG)_CHECKSUM := 686affff989ac2488f79a97b9479efb9f2abae035b5ed4d8226de6857933fd3b
+$(PKG)_VERSION  := 1.71.0
+$(PKG)_CHECKSUM := 96b34f7468f26a141f6020efb813f1a2f3dfb9797ecf76a7d7cbd843cc95f5bd
 $(PKG)_SUBDIR   := boost_$(subst .,_,$($(PKG)_VERSION))
-$(PKG)_FILE     := boost_$(subst .,_,$($(PKG)_VERSION)).tar.bz2
-$(PKG)_URL      := https://$(SOURCEFORGE_MIRROR)/project/boost/boost/$($(PKG)_VERSION)/$($(PKG)_FILE)
+$(PKG)_FILE     := boost_$(subst .,_,$($(PKG)_VERSION)).tar.gz
+$(PKG)_URL      := https://dl.bintray.com/boostorg/release/$($(PKG)_VERSION)/source/$($(PKG)_FILE)
 $(PKG)_TARGETS  := $(BUILD) $(MXE_TARGETS)
-$(PKG)_DEPS     := cc bzip2 expat zlib
+$(PKG)_DEPS     := cc bzip2 expat zlib xz
+AWK             = $(shell which $(shell gawk --help >/dev/null 2>&1 && echo g)awk)
```

Comment:
Add such variables near the [top of the main Makefile](https://github.com/mxe/mxe/blob/master/Makefile#L32).

Reply:
Can't fix it until tomorrow but will do.